### PR TITLE
Clin-163:: cpt to procedure

### PR DESCRIPTION
**Pull request by arslansajid**

### CODE REVIEW

File: server/app/controllers/procedure.controller.js [hunk 1]
```diff
@@ -90,30 +90,26 @@ const search = async (req, res) => {
...
+      } /*TODO if fee is "" then set fee to null*/, '${notes}', now(), ${req.user_id
+      }, now(), ${req.user_id} ) 
          on duplicate key update 
             favorite=${favorite},
             billable=${billable},
-            fee=${
-              fee > 0 ? fee : 0
-            },  /*TODO if fee is "" then set fee to null*/
+            fee=${fee > 0 ? fee : 0
+      },  /*TODO if fee is "" then set fee to null*/
```
Comment (avastamin) [inline server/app/controllers/procedure.controller.js:112]: Still `TODO` should be here?

**Review by avastamin — APPROVED** [review]

File: server/app/controllers/procedure.controller.js (hunk 1 above)
Comment (arslansajid) [reply server/app/controllers/procedure.controller.js:112]: I have worked on the renaming of **cpt** to **procedure**. I am not sure about the functionality of this method. We can update the logic after discussing it with @davidjhp .

File: client/src/screens/Client/Setup/ProcedureCodes/component/ProcedureTable.js [hunk 2]
```diff
@@ -60,7 +60,7 @@ const StyledTableRow = withStyles((theme) => ({
   },
 }))(TableRow);
 
-const CPTtable = ({ searchResult, fetchCptCodeSearch }) => {
+const Proceduretable = ({ searchResult, fetchProcedureCodeSearch }) => {
```
[the remark this answers is not in the document]
Comment (arslansajid) [reply client/src/screens/Client/Setup/ProcedureCodes/component/ProcedureTable.js:63]: Done.

File: client/src/screens/Client/Setup/ProcedureCodes/component/ProcedureForm.js [hunk 3]
```diff
@@ -37,23 +37,17 @@ const useStyles = makeStyles(() => ({
   },
 }));
 
-const CPTform = ({
+const Procedureform = ({
```
[the remark this answers is not in the document]
Comment (arslansajid) [reply client/src/screens/Client/Setup/ProcedureCodes/component/ProcedureForm.js:40]: Done.

**Pull request merged**